### PR TITLE
[MM-47818] userInviteCmdF function to return an error in case of a failure

### PR DESCRIPTION
**Pull request by ellisonleao**
#### Summary

Added multi error handing in userInviteCmdF function

#### Ticket Link

Fixes: https://github.com/mattermost/mattermost-server/issues/21468
Jira: https://mattermost.atlassian.net/browse/MM-47818

### CODE REVIEW

File: commands/user.go [hunk 1]
```diff
@@ -11,8 +11,7 @@ import (
 
 	"github.com/hashicorp/go-multierror"
 
-	"github.com/mattermost/mattermost-server/v6/model"
-
+  "github.com/mattermost/mattermost-server/v6/model"
```
Comment (noxer) [inline commands/user.go:14]: The linter complains that the imports are not in the right order.

**Review by noxer — CHANGES_REQUESTED** [review]
Hi @ellisonleao, thank you very much for your contribution. I'd like to suggest a few improvements and the linter complains about the order of the imports.

File: commands/user.go (hunk 1 above)
Comment (ellisonleao) [reply commands/user.go]: done

File: commands/user_test.go [hunk 2]
```diff
@@ -1184,7 +1184,7 @@ func (s *MmctlUnitTestSuite) TestUserInviteCmd() {
 			Times(1)
 
 		err := userInviteCmdF(s.client, &cobra.Command{}, append([]string{argUser}, argTeam...))
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```
[the remark this answers is not in the document]
Comment (ellisonleao) [reply commands/user_test.go]: done

File: commands/user_test.go [hunk 3]
```diff
@@ -1076,7 +1076,7 @@ func (s *MmctlUnitTestSuite) TestUserInviteCmd() {
 			Times(1)
 
 		err := userInviteCmdF(s.client, &cobra.Command{}, []string{argUser, argTeam})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```
[the remark this answers is not in the document]
Comment (ellisonleao) [reply commands/user_test.go]: done

File: commands/user_test.go [hunk 4]
```diff
@@ -1050,7 +1050,7 @@ func (s *MmctlUnitTestSuite) TestUserInviteCmd() {
...
```
[the remark this answers is not in the document]
Comment (ellisonleao) [reply commands/user_test.go]: done

**Review by noxer — APPROVED** [review]
Thank you @ellisonleao, this looks good to me.

**Review by isacikgoz — APPROVED** [review]
Thanks for the contribution @ellisonleao LGTM!

**Pull request merged**